package.json

```diff
@@ -2,5 +2,8 @@
   "dependencies": {
     "bootstrap": "^5.2.3",
     "vue": "^3.2.47"
+  },
+  "devDependencies": {
+    "standard": "^17.0.0"
   }
-}
+}
```

Comment:
Newline is missing, I think there should be one at the end of the file

Reply:
Added